sys/sys_amd64.const

```diff
@@ -200,6 +200,7 @@ KEXEC_ARCH_SH = 2752512
 KEXEC_ARCH_X86_64 = 4063232
 KEXEC_ON_CRASH = 1
 KEXEC_PRESERVE_CONTEXT = 2
+KSTAT_ATTR_FS_IOC_FLAGS = 2164
```

Comment:
Please generate constants for other architectures as well. You need to install cross compilers for that.